fedimint-recoverytool/src/key.rs

```diff
@@ -13,6 +13,10 @@ pub enum Key {
     Private(bitcoin::key::PrivateKey),
 }
 
+// TODO:
```

Comment:
@m1sterc001guy clippy correctly flags now that PartialOrd and Ord work differently. Thoughts on changing PartialOrd to just use Ord (`Some(self.cmp(other))`)?